salt/modules/crmsh.py

```diff
@@ -9,3 +9,108 @@
 :depends: crmsh
 '''
 from __future__ import absolute_import
+import re
+
+from salt.ext import six
+from salt.exceptions import CommandExecutionError
+
+
+def _dc():
+    '''
+    check whether we are DC node
+    '''
+    import socket
+
+    try:
+        res = __salt__['cmd.run_all']("crmadmin -D")
+    except:
+        return False
+    if 'stdout' in res.keys():
+        dc = res['stdout'].split()[-1]
+        if dc != socket.gethostname():
+            return False
+
+    return True
+
+
+def configure_property(prop):
```

Comment:
This is a bit awkward, would it not be possible to take `key` and `value` as arguments?

Reply:
configure_property(key, value) will be clear,
but in this way, I can set more pairs of k-v, like prop="key1=value1 key2=value2 key3=value3" ,
otherwise, use **keywords parameters will be better?